### PR TITLE
Lablr fixey boi

### PR DESCRIPTION
**Pull request by ALEXWARELLC**

### CODE REVIEW

File: Exiled.API/Enums/EffectType.cs [hunk 1]
```diff
@@ -230,9 +230,14 @@ public enum EffectType
         Ghostly,
 
         /// <summary>
-        /// Manipulate which fog type the player will have.
+        /// Manipulate wish Fog player will have.
```
Comment (Tinoyeet) [inline Exiled.API/Enums/EffectType.cs:233]: isn't this wrong?

Comment (ALEXWARELLC) [reply Exiled.API/Enums/EffectType.cs:233]: u on something?

Comment (ALEXWARELLC) [reply Exiled.API/Enums/EffectType.cs:233]: why are you here?

Comment (Tinoyeet) [reply Exiled.API/Enums/EffectType.cs:233]: i got lost

**Pull request closed without merging**